modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/SnapshotPartitionsVerifyHandler.java

```diff
@@ -362,6 +362,13 @@ private Map<PartitionKeyV2, PartitionHashRecordV2> checkDumpFiles(
         SnapshotHandlerContext opCtx,
         Set<File> partFiles
     ) {
+        if (F.isEmpty(partFiles)) {
+            log.info("Dump data integrity check skipped [snpName=" + opCtx.metadata().snapshotName()
```

Comment:
snpName -> dumpName, let's also add param for "this node" - consistentId?.

Reply:
Thought about it. All the other code uses 'snp' and 'snapshot' for dumps. But ok